linkcheck/htmlutil/linkparse.py

```diff
@@ -99,42 +99,44 @@ class StopParse(Exception):
 
 class TagFinder (object):
     """Base class handling HTML start elements.
-    TagFinder instances are used as HtmlParser handlers."""
+    TagFinder instances are used as HTML parser handlers."""
 
     def __init__ (self):
         """Initialize local variables."""
         super(TagFinder, self).__init__()
-        # parser object will be initialized when it is used as
-        # a handler object
-        self.parser = None
+        # Default to handling all tags
+        self.tag_filter = True
 
-    def start_element (self, tag, attrs):
+    def start_element (self, tag, attrs, element_text, lineno, column):
         """Does nothing, override in a subclass."""
         pass
 
-    def start_end_element (self, tag, attrs, element_text=None):
+    def start_end_element (self, tag, attrs, element_text, lineno, column):
         """Delegate a combined start/end element (eg. <br/>) to
         the start_element method. Ignore the end element part."""
-        self.start_element(tag, attrs, element_text)
+        self.start_element(tag, attrs, element_text, lineno, column)
 
 
 class MetaRobotsFinder (TagFinder):
     """Class for finding robots.txt meta values in HTML."""
+    TAG_BODY = "body"
+    TAG_META = "meta"
 
     def __init__ (self):
         """Initialize follow and index flags."""
         super(MetaRobotsFinder, self).__init__()
         log.debug(LOG_CHECK, "meta robots finder")
         self.follow = self.index = True
+        self.tag_filter = MetaRobotsFinder.TAG_BODY, MetaRobotsFinder.TAG_META
```

Comment:
```suggestion
        self.tag_filter = (MetaRobotsFinder.TAG_BODY, MetaRobotsFinder.TAG_META)
```

Reply:
The change is now not included in this PR, a fixed version will be in the next.